modelopt/torch/speculative/plugins/transformers.py

```diff
@@ -772,6 +786,8 @@ def forward(
         loss_mask: torch.Tensor | None = None,
         classification_loss_coefficient: float | None = 1,
         regression_loss_coefficient: float | None = 0,
+        hidden_states: torch.Tensor | None = None,
+        aux_hidden_states: list[torch.Tensor] | None = None,
```

Comment:
In the latest main branch, these two optional inputs are put into kwargs. I think the logic are the same. Could you please rebase onto main branch to solve the conflict?